### ISSUE TITLE
WiGLE compatibility

### ISSUE THREAD

**Issue by Yoanndp** [mjaakko/NeoStumbler#113]
Hello, is it possible to export data in a way that is compatible with WiGLE data structure ?

**Comment by mjaakko** [comment]
Right now it's not possible. The export format is custom to NeoStumbler. It's just simple CSV files inside a ZIP, so it should be simple to work with

I'll think about adding support for exporting data in more standardized data formats, such as the one used by Wigle. I'd be also happy to accept a PR, if you need this feature urgently

**Comment by mjaakko** [comment]
Yes, you can keep this issue open for discussions and tracking the progress
